components/odh-notebook-controller/e2e/notebook_update_test.go

```diff
@@ -65,7 +65,9 @@ func (tc *testContext) testNotebookUpdate(nbContext notebookContext) error {
 	}
 
 	// Example update: Change the Notebook image
-	updatedNotebook.Spec.Template.Spec.Containers[0].Image = "new-image:latest"
+	// newImage := "new-image:latest"  quay.io/thoth-station/s2i-minimal-notebook:v0.2.2
```

Comment:
I sort of like that the test uses image that does not exist. Would change it to example.org/new-image:latest, to be sure it really does not exist, maybe?

Reply:
I don't follow. My expectation is that without a working image link, the test would fail :thinking:

Reply:
the way you wrote it now, with `waitForStatefulSet`, yes; but the original intention in the test I think was to not actually have the updated notebook running...

let's have it running; I don't particularly mind! the previous design (at least how I imagine the previous design to be) would be tricky because it's not immediately clear what can and what cannot be asserted

Reply:
Using the illegal image link would result into these failures:
```
            --- FAIL: TestE2ENotebookController/update/thoth-minimal-oauth-notebook/Notebook_Statefulset_Validation_After_Update (60.34s)
            --- FAIL: TestE2ENotebookController/update/thoth-minimal-oauth-notebook/Notebook_OAuth_sidecar_Validation_After_Update (0.14s)
            --- FAIL: TestE2ENotebookController/update/thoth-minimal-oauth-notebook/Verify_Notebook_Traffic_After_Update (10.94s)
```

So, I don't think it was the intention - mainly the traffic sounds like there should be something running.